File: packages/server/src/infrastructure/__fixtures__/sample.php
<?php

const ORIGIN = [0, 0];

function distance($a, $b) {
    return sqrt(pow($a[0] - $b[0], 2) + pow($a[1] - $b[1], 2));
}

class Vec {
    public function __construct(public float $x, public float $y) {}

    public function length(): float {
        return sqrt($this->x * $this->x + $this->y * $this->y);
    }
}

File: packages/server/src/infrastructure/__fixtures__/sample.py
ORIGIN = (0, 0)


def distance(a, b):
    return ((a[0] - b[0]) ** 2 + (a[1] - b[1]) ** 2) ** 0.5


class Vec:
    def __init__(self, x, y):
        self.x = x
        self.y = y

    def length(self):
        return (self.x ** 2 + self.y ** 2) ** 0.5
